pkg/ebpf/tracer-ebpf.go

```diff
@@ -111,7 +111,7 @@ func tracerEbpfDebugO() (*asset, error) {
70\x6c\x3c\x6a\x50\xda\x03\xb3\x7b\xe9\x2d\xb3\xb2\x3c\x83\x91\xbc\x68\xd3\x27\xad\xf3\xd0\x62\x78\xac\xba\xa5\x9f\x22\x3b\x26\x1a\x48\xb6\x7c\x93\x5d\x0b\xf0\xcf\xfb\x7f\x8d\x68\x5e\x11\x85\x25\x3f\x58\xb3\x1d\x26\xe7\x6d\x1c\xac\xb9\x18\x39\x58\xd3\x59\xe9\xc3\x30\x81\x1f\x0c\xd0\xb1\x95\xb9\x54\x51\xdd\x2d\x0b\xb5\x1d\x69\xcc\x5a\x29\xd4\x60\xa3\x50\xc1\xc2\xc1\x9a\x64\x57\xa8\x2b\x78\x00\xae\x95\xa3\x93\x8f\x59\x58\x2a\x56\xe1\x76\x32\x3b\x0b\x4e\x6a\x03\x3c\x0d\xc8\x40\xa1\x92\x5d\xf8\xbf\x36\x00\x70\x75\x66\x3b\x6f\x17\x77\x83\xbf\xe9\x04\x6d\x2a\x8e\x57\x10\x3e\x53\x4e\xcf\xd1\xdc\x7f\xff\xf7\xdc\x1d\x90\xd1\xc5\xb9\x2e\x86\x43\x17\x02\x4b\x69\xe0\xa1\x1b\x17\x0b\x57\x74\x06\xae\xfd\x24\x32\x5a\xa9\xb5\x63\x42\xd8\x09\xc5\x41\x4a\x9e\x4b\x93\xec\x5a\xac\x20\xf9\x75\x2c\x31\x8b\x25\x6e\xb0\xc4\xef\xc0\x72\x46\x5b\x7f\xcb\x8b\xa4\x53\x5e\x41\xf2\x1b\xbc\x66\xb1\xc4\x0d\x96\x18\xb0\x42\x88\x27\x01\x0e\xdd\x42\x88\xaf\xa7\xf8\x86\x61\x17\xde\x08\xa0\x7e\xf5\x12\x2c\x3f\xa6\xea\xe4\x57\xc0\xb5\x91\x28\x52\x87\x1f\x56\xed\x69\xde\xfe\x3a\x25\xd2\x5a\x31\x82\xbf\x26\x3a\x2c\xa5\x4d\xd1\xb2\xec\x44\x71\x90\xdd\x79\x93\xec\x3a\x09\x00\xc4\xdb\x70\x3c\xfc\xd2\x35\x01\x0d\x53\xbe\xbd\x45\x42\x73\x04\x0a\x4f\x7e\xc6\xc8\x98\x56\xbf\x45\x07\x11\xc5\x81\x8c\x8c\x24\xad\x91\x37\x74\x0d\xe8\x0c\xce\xe8\xa1\x56\xe1\x52\x10\xcc\xb3\x19\xfc\x91\xab\x3d\xfe\x20\x22\x37\x02\xfe\x20\xe9\x9d\x78\xcf\x64\xb9\xf8\xf3\x97\xa1\xe7\x24\x60\xc8\xb1\x14\xa1\x7f\xe3\x06\x46\x4d\xe9\x8c\xc1\x2e\xa0\xe3\xa8\x75\xa2\x51\xd4\x3a\xc9\x25\x6a\xdf\x87\x23\x73\x7f\xf6\x38\xc4\x8e\xfa\x2a\x5e\x40\xa6\x2d\xe5\x92\x9a\xa5\xa1\xe3\xe9\x83\x9e\x8c\xb2\xda\x37\x19\x9d\xd2\x6b\xba\xc2\x86\x05\xf7\xdb\x0e\x62\xe8\x7c\x2e\x2e\x27\x97\x1a\x98\x73\x27\xd9\xb5\x28\x6b\x72\x66\x5c\xae\x23\x31\xe5\x89\xae\x15\x57\xa4\x96\x1f\xf7\x75\x16\xcf\xd5\x93\x4b\x76\xcd\x3c\x54\x33\x0b\xe5\x94\x68\xa1\xa6\x9e\xed\xe0\x62\xbc\x62\xa7\xb8\xbf\xa6\x2f\x22\x8a\x50\x18\xdc\xc4\x64\xa4\x26\xd5\xcd\xca\xd0\x03\x4e\x57\x36\x23\x9b\xcd\xad\xcd\x66\x6c\xb3\x99\x5b\xf9\x7a\xac\x36\xb2\xd9\x8c\x6d\xf6\x2b\x23\x18\x35\x86\xcf\xa3\x26\x5d\xb4\xb7\x2d\xa7\x26\x3d\x74\x59\xac\xc2\x71\xef\x1c\x43\x56\x2b\x1e\x7a\x1b\xa9\x2e\x0f\x8c\x75\x40\xc8\x91\x50\xad\xc4\x23\x2b\xc3\x93\x23\x47\x3a\x45\x14\x7c\xa8\x99\xb0\xb5\x29\xd1\x87\x2e\x7e\x8f\x20\xa8\xd3\x0c\x9d\x3d\xbd\x45\x38\xe3\x39\x8a\xfe\xd9\x52\x31\x13\x51\x09\x15\x6d\x4e\xd3\xd1\xcb\xa5\x6f\x6b\x8c\x4f\x2d\x1e\x1c\x2c\xb9\x3f\xc5\xd1\xb8\x7f\xef\x9a\xfa\x51\x0b\xd4\x3d\x45\xb8\x3f\x25\x3b\x21\xdb\xf3\xea\x9b\xaf\xa0\x6f\xab\x7e\xb5\xe3\xa2\x18\x8f\x9e\x43\x6b\xa9\x1c\x5a\xbf\xb8\xeb\x35\x42\x0d\xc5\xe3\x78\x8e\xdf\x19\x34\xf3\x1c\xb5\xbe\xf4\x91\xfa\x9d\xee\xfc\x4f\x0c\x27\x8e\xb6\xce\x8f\x49\xb7\xef\x85\x0b\xe9\x1b\xaf\x04\x96\xe8\x71\xa2\xd0\xf7\xa9\x45\x5b\x47\xb7\xce\x2d\xbb\xab\x75\x52\x46\xd7\x59\xeb\x9e\x86\x7f\x54\xd6\xfa\xe7\xed\xe7\xca\xda\x6f\xba\xf3\x39\xb2\xd6\x93\xfe\x64\xd6\x7a\x85\xcf\x95\xb5\xff\x06\x00\x00\xff\xff\x33\xd6\x21\xf8\x39\x12\x00\x00")
```

Comment:
This should not have changed since you didn't touch any eBPF code, you may need to re-build your dev VM to pull in the right version of clang.

Reply:
I rebuilt my dev VM -  but this modification is still happening. Any suggestion on what I should check next?

Reply:
It looks like only the tcpQueueLength program is changing now.  I think someone managed to check in a version built with an older version of clang because we don't have a test to prevent it the way we do with the older eBPF program. In that case, I think it's fine to merge this.